### DISCUSSION TITLE
moduel 'requests' , 'pip' installation issues

### DISCUSSION THREAD

**Discussion by RobHaydon92** [itsToggle/plex_debrid#162] [Q&A]
Hi,
Im trying to run the script in python but am getting this error,

python error: (module import exception):
No module named 'requests'
Make sure you have installed this python module.
You need to install 'pip' (https://pip.pypa.io/en/stable/installation/) and run the command 'pip install requests'.
Press any key to exit

I have installed the moduel 'requests' and 'pip' and both show as installed and working but when running your script I conue to receve this issue, is there something im missing or is it a version support issue im encountering
im running python 3.10

Thanks

**Comment by itsToggle** [comment]
sounds like you installed pip and requests to python 2.XX and not python 3 :)

try running pip3 install ....

**Reply by RobHaydon92** [reply]
Thanks a bunch managed to sort this,
